lib/LMSManagers/LMSNetworkManager.php

```diff
@@ -204,7 +204,7 @@ public function GetNetIDByIP($ipaddr)
 
     public function GetUnlinkedNodes()
     {
-        return $this->db->GetAll('SELECT n.*, inet_ntoa(n.ipaddr) AS ip, net.name AS netname
+        return $this->db->GetAll('SELECT n.id, n.name, inet_ntoa(n.ipaddr) AS ip, net.name AS netname
```

Comment:
1. To naprawdę zwiększa wydajność?
2. Inne właściwości z nodes n nie są nigdzie używane po wywołaniu tej metody?

Reply:
1. Jasne! I to znacząco. Z 40Mb wartości generowanej strony zrobiło się 5Mb.
2. Sprawdziłem wywołania funkcji i w GUI są potrzebne tylko te dwie wartości w liście wyboru niepodpiętych komputerów.

Reply:
2. A w innych miejscach ta funkcja nie jest używane?

Reply:
Jest ale dokładnie w ten sam sposób.